docs/source/NEWS.rst

```diff
@@ -2,7 +2,7 @@ Changelog
 =========
 
 
-Release 18.0 (UNRELEASED)
+Release 19.0 (2019-03-05)
```

Comment:
the release date is set, but I don't see the file on PyPI

Reply:
Because I haven't uploaded it yet. ;)